src/NBitcoin/Consensus.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using NBitcoin.BouncyCastle.Math;
+
+namespace NBitcoin
+{
+    public enum BuriedDeployments : int
+    {
+        /// <summary>
+        /// Height in coinbase
```

Comment:
Dots in the summaries of this enum are missing.

Reply:
👍